build-tools/push-federation-images.sh

```diff
@@ -22,8 +22,5 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
-source "${KUBE_ROOT}/build-tools/util.sh"
-
-source "${KUBE_ROOT}/federation/cluster/common.sh"
-
-FEDERATION_IMAGE_TAG="$(kube::release::semantic_image_tag_version)" push-federation-images
```

Comment:
Yeah, but let's do that in stages. Just in case if we need that to debug or anything.